force-app/main/default/classes/newTaskTraining.cls

```diff
@@ -0,0 +1,18 @@
+public class newTaskTraining {
+    public static void createTask() {
+        Date today = Date.today();
+        List<Training__c> endedTrains = [SELECT Name, End_Date__c, Training_Course__r.OwnerId
+                                        FROM Training__c
+                                        WHERE DAY_ONLY(End_Date__c) = :today];
+        //System.debug(endedTrains);
+        List<Task> taski = new List<Task>();
```

Comment:
"taski" is a strange naming, because it doesn't explain anything about the purpose of the variable. I would name it taskList or tasksToInsert. It is considered as a best practice to have names variables in such a way that after only reading a name of the variable you can recognize what is its purpose.